src/wallet/rpcwallet.cpp

```diff
@@ -181,6 +181,7 @@ static void WalletTxToJSON(interfaces::Chain& chain, const CWalletTx& wtx, UniVa
             rbfStatus = "yes";
     }
     entry.pushKV("bip125-replaceable", rbfStatus);
+    entry.pushKV("inmempool", wtx.fInMempool);
```

Comment:
`"in_mempool"` seems like a better name.

Reply:
:+1: for `in_mempool` ( we use snakecased field names now in the JSON RPC output)